### PR TITLE
feat: add force flag to checkpoint command

### PR DESCRIPTION
**Pull request by wadealexc**
* also tweaks logging and flag names
* also fixes 2 bugs when filtering validators for a checkpoint

### CODE REVIEW

File: cli/utils.go [hunk 1]
```diff
@@ -47,22 +53,31 @@ type Owner = struct {
...
 	PanicOnError("failed to parse private key", err)
 
 	eigenPod, err := onchain.NewEigenPod(gethCommon.HexToAddress(eigenpodAddress), eth)
 	PanicOnError("failed to reach eigenpod", err)
 
-	txn, err := eigenPod.StartCheckpoint(ownerAccount.TransactionOptions, true)
+	// revertIfNoBalance == !forceCheckpoint
+	// The CLI exposes this as a `force` parameter for usability
+	revertIfNoBalance := true
+	if forceCheckpoint {
```
Comment (jbrower95) [inline cli/utils.go:66]: `revertIfNoBalance := !forceCheckpoint`

File: cli/utils.go [hunk 2]
```diff
@@ -162,8 +177,9 @@ func FilterNotCheckpointedOrWithdrawnValidators(
 
 		notCheckpointed := validatorInfo.LastCheckpointedAt != lastCheckpoint
 		notWithdrawn := validatorInfo.Status != ValidatorStatusWithdrawn
+		notInactive := validatorInfo.Status != ValidatorStatusInactive
 
-		if notCheckpointed && notWithdrawn {
+		if notCheckpointed && notWithdrawn && notInactive {
```
Comment (jbrower95) [inline cli/utils.go:182]: do we want to just explicitly check `validatorInfo.Status == ValidatorStatusActive` instead?

Comment (wadealexc) [reply cli/utils.go:182]: You could combine the last 2 into a `isActive` flag if you want but idc personally

Comment (wadealexc) [reply cli/utils.go]: jk this is nicer i change mind

**Review by jbrower95 — APPROVED** [review]

**Pull request merged**